src/custom-elements/src/components/tabs/tabs.tsx

```diff
@@ -352,11 +350,32 @@ export class Tabs {
     this.isSeeMoreActive = false;
   }
 
-  isActiveTabOverflown() {
-    return (
-      this.tabs.findIndex(tab => tab.id === this.activeTab && tab.overflow) !==
-      -1
-    );
+  _isActiveTabChildren(tab: TabTrigger) {
+    if (tab.id === this.activeTab) {
+      return true;
+    } else if (tab.children) {
+      return this.isActiveTabOverflown(tab.children, true);
+    }
+  }
+
+  _isActiveTabParent(tab: TabTrigger) {
+    if (tab.id === this.activeTab && tab.overflow) {
+      return true;
+    } else if (tab.children) {
+      return this.isActiveTabOverflown(tab.children, true) && tab.overflow;
+    }
+  }
+
+  isActiveTabOverflown(tabs: TabTrigger[], isChildren: boolean): boolean {
+    const isSomeTabActive = (tabs: TabTrigger[]): boolean => {
+      return tabs.some(tab =>
+        isChildren
+          ? this._isActiveTabChildren(tab)
+          : this._isActiveTabParent(tab)
```

Comment:
With new approach this will look like this:
```
this._isActiveTab(tab, !isChildren)
```
And we could even replace isChildren by isParent so every method accepts the same parameters